Makefile

```diff
@@ -6,7 +6,7 @@ CURRENT_DATE = $(strip $(shell date -u +"%Y-%m-%dT%H:%M:%SZ"))
 CODE_VERSION = $(strip $(shell cat VERSION))
 FULL_IMAGE_NAME = $(strip $(IMAGE_NAME):$(CODE_VERSION))
 
-PLATFORMS = linux/amd64,linux/arm/v7
+PLATFORMS = linux/amd64,linux/arm/v7,linux/arm64/v8
```

Comment:
This should also be included in the following
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/pr.yml#L93
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/release.yml#L94
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/stage.yml#L66